samples/mysql-schema/src/main/java/io/quarkiverse/operatorsdk/samples/mysqlschema/schema/Schema.java

```diff
@@ -40,4 +42,9 @@ public int hashCode() {
     public String toString() {
         return "Schema{" + "name='" + name + '\'' + ", characterSet='" + characterSet + "'}";
     }
+
+    @Override
+    public String resourceId() {
+        return getName();
```

Comment:
this could just directly return field but no biggie

Reply:
Then maybe add a comment with this? I think it might be important for someone checking this sample.